public/pages/Dashboard/Components/EmptyDashboard/EmptyDashboard.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import {
+  EuiButton,
+  EuiEmptyPrompt,
+  EuiLink,
+  EuiIcon,
+  EuiPage,
+  EuiPageBody,
+} from '@elastic/eui';
+import React, { Component, Fragment } from 'react';
+import { APP_PATH, PLUGIN_NAME } from '../../../../utils/constants';
+import { SideBar, DashboardHeader } from '../utils/common';
+
+export class EmptyDashboard extends Component<{}, EmptyDashboardState> {
+  render() {
+    return (
+      <EuiPage style={{ flex: 1 }}>
+        <SideBar />
+        <EuiPageBody>
+          <DashboardHeader />
```

Comment:
This component should be just EmptyDashboard, I would suggest you to move rest of the things to actual Page.

Reply:
Moved.